uber/config.py

```diff
@@ -65,8 +65,7 @@ def make_enum(self, enum_name, section, prices=False):
                     val = name
             else:
                 varnames.append(name.upper())
-                val = int(sha512(name.upper().encode()).hexdigest()[:7], 16)
-                setattr(self, name.upper(),  val)
+                self.create_enum_val(name)
```

Comment:
I'm pretty sure you need to say ``val = self.create_enum_val(name)`` here, since otherwise you're not setting ``val`` to the correct thing; the value of ``val`` is referenced below, so if you don't set it here then we'll be pushing the wrong thing to our ``_OPTS`` list and such.